### PR TITLE
Migrating web_analysis from Cirrus to LUCI (PART 1)

### PR DESCRIPTION
**Pull request by nturgut**
adding a script for web_analysis for using from luci.

PART1: Add script to run web_analysis.
PART2: Change LUCI recipe to run web_analysis checks (Successful run example with the recipe change: https://chromium-swarm.appspot.com/task?id=4d631dfb7ab22610)
PART3: Cleanup this step from Cirrus file.

### CODE REVIEW

File: lib/web_ui/dev/web_engine_analysis.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
```
Comment (yjbanov) [inline lib/web_ui/dev/web_engine_analysis.sh:2]: `set -x` would also be useful

File: lib/web_ui/dev/web_engine_analysis.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# web_analysis: a command-line utility for running dart analyzer on Flutter Web
+# Engine. Used/Called by LUCI recipes:
+#
+# See: https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/web_engine.py#165
```
Comment (yjbanov) [inline lib/web_ui/dev/web_engine_analysis.sh:7]: The pointer to the line number 165 will get out-of-sync. I'd link to just the file, without the line number.

Comment (nturgut) [reply lib/web_ui/dev/web_engine_analysis.sh:7]: makes sense, thanks!

File: lib/web_ui/dev/web_engine_analysis.sh (hunk 1 above)
Comment (nturgut) [reply lib/web_ui/dev/web_engine_analysis.sh:2]: done

**Comment by nturgut** [comment]
Merging the PR since the failing test (seems to be flaky) is not on the code path.

**Pull request merged**